GPU/Workflow/src/GPUWorkflowSpec.cxx

```diff
@@ -133,6 +135,49 @@ void GPURecoWorkflowSpec::init(InitContext& ic)
   GRPGeomHelper::instance().setRequest(mGGR);
   GPUO2InterfaceConfiguration& config = *mConfig.get();
 
+  if (mNNClusterizerSettings->nnLoadFromCCDB) {
+    LOG(info) << "Loading neural networks from CCDB";
+    o2::tpc::NeuralNetworkClusterizer nnClusterizerFetcher;
+    nnClusterizerFetcher.initCcdbApi(mNNClusterizerSettings->nnCCDBURL);
+    std::map<std::string, std::string> ccdbSettings = {
+      {"nnCCDBURL", mNNClusterizerSettings->nnCCDBURL},
```

Comment:
Strictly speaking, all CCDB-related settings so far are in `GPUSettingsO2`, not in `GPUSettingsProcessing`. Perhaps it would be cleaner to also move the NN CCDB settings there, but since they are anyway encapsulated in your own struct, I don't care much.